templates/repo/home.tmpl

```diff
@@ -124,7 +124,7 @@
 				{{if eq $n 0}}
 					<div class="ui action tiny input" id="clone-panel">
 						{{template "repo/clone_buttons" .}}
-						<button id="download-btn" class="ui basic jump dropdown icon button tooltip" data-content="{{.locale.Tr "repo.download_archive"}}" data-position="top right">
+						<button id="download-btn" class="ui basic jump dropdown icon button tooltip" data-content="{{.locale.Tr "repo.download_archive"}}" data-position="top right" aria-label="{{.locale.Tr "repo.download_archive"}}">
```

Comment:
This button would change to a regular menu with https://github.com/go-gitea/gitea/pull/19999, so I'm not sure how much sense it makes to create this future merge conflict now :)

Reply:
Yeah, keep it in 1.18. I expect that PR to land soon to 1.18.

Reply:
k, updated :)